packages/backend/Dockerfile

```diff
@@ -1,79 +1,78 @@
 # From https://github.com/janus-idp/backstage-showcase/
-# Stage 1 - Install dependencies
-FROM registry.access.redhat.com/ubi9/nodejs-18:latest AS deps
+# Stage 1 - Build nodejs skeleton
```

Comment:
I used the [janus show case docker file at the v 1.15 upgrade](https://github.com/janus-idp/backstage-showcase/blob/5d983a537c95325a33584311bb26436d80214ef0/Dockerfile) as the basis for this docker file. It's pretty much the same except for adding in plugins and removing techdocs dependencies as techdocs are external for us